fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -292,7 +292,7 @@ def build_xcodebuild_showbuildsettings_command
     end
 
     # Get the build settings for our project
-    # this is used to properly get the DerivedData folder
+    # this is used e.g. to properly get the DerivedData folder
```

Comment:
I think this line should be reworded to just say `e.g. to properly get the DerivedData folder`